pkg/apiserver/registry/generic/key.go

```diff
@@ -44,8 +44,22 @@ func ParseKey(raw string) (*Key, error) {
 			key.Namespace = v
 		case "name":
 			key.Name = v
+		case "pods":
```

Comment:
🤣 obviously not a real solution... but this lets most of the tests get closer to running.  There must be some better way, but this lets us focus on the behavior first then the keys